app/src/test/java/comcmput301f16t01/github/carrier/UserProfileTests.java

```diff
@@ -46,21 +46,21 @@ public void uniqueDriverUsername() {
      */
     @Test
     public void editRiderInfo() {
-        Rider riderOne = new Rider("start username");
-        riderOne.setEmail("start email");
-        riderOne.setPhone("start phone");
+        UserController uc = new UserController();
+        uc.getLoggedInUser().setEmail("start email");
+        uc.getLoggedInUser().setPhone("start phone");
 
-        assertTrue(riderOne.getUsername().equals("start username"));
-        assertTrue(riderOne.getEmail().equals("start email"));
-        assertTrue(riderOne.getPhone().equals("start phone"));
+        assertTrue(uc.getLoggedInUser().getUsername().equals("start username"));
+        assertTrue(uc.getLoggedInUser().getEmail().equals("start email"));
+        assertTrue(uc.getLoggedInUser().getPhone().equals("start phone"));
```

Comment:
(No issue here!) These assert statements work, but I would have taken the ones you deleted as well. (i.e. `assertTrue(riderOne.getUsername().equals("start username"));`). The only problem was that you didn't use a controller to modify the model and you fixed that! 👍